### DISCUSSION TITLE
Collection filters Facets are NOT translated

### DISCUSSION THREAD

**Discussion by bitforcesrl** [Shopify/storefront-api-feedback#166] [Help]
Hi,
Is there a way to retrieve the translated version of filters?
https://shopify.dev/api/storefront/unstable/objects/FilterValue

I'm receiving the default language values also passing language and country through inContext.

```
${PRODUCT_CARD_FRAGMENT}
query CollectionDetails(
  $handle: String!
  $country: CountryCode
  $language: LanguageCode
  $pageBy: Int!
  $cursor: String
  $productFilters: [ProductFilter!] = []
  $sortKey: ProductCollectionSortKeys
  $reverse: Boolean = false
) @inContext(country: $country, language: $language) {
  collection(handle: $handle) {
    id
    title
    description
    seo {
      description
      title
    }
    image {
      id
      url
      width
      height
      altText
    }
    products(
      first: $pageBy
      after: $cursor
      filters: $productFilters
      sortKey: $sortKey
      reverse: $reverse
    ) {
      nodes {
        ...ProductCard
      }
      filters {
        id
        label
        type
        values {
          id
          label
          count
          input
        }
      }
      pageInfo {
        hasNextPage
        endCursor
      }
    }
  }
}
```

Thank you

**Comment by bitforcesrl** [comment]
Hi @vixdug we've also opened this issue that is similar to the menu, also filter facets are not translated this means that we retrieve variants options in the main store language and not in the current user selected language. this is a blocking issue for us at the moment.
any feedback will be appreciated. thanks

**Comment by jvmiert** [comment]
Also facing this issue. The filter labels itself are correctly returning the translated values however the filter value labels are not.
